demo/src/pages/Workout.tsx

```diff
@@ -527,13 +529,64 @@ const Workout = ({ history }) => {
     isPreJoin,
   };
 
+  const [connectionDetails, setConnectionDetails] = useState() as any;
```

Comment:
This code probably should go inside the SDK. If we add in demo app, client may skip adding this.